app/Domain/Tickets/Templates/components/milestone-dropdown.blade.php

```diff
@@ -35,6 +36,7 @@
                     data-value="{{ $ticket['id'] . '_' . $milestone->id . '_' . $milestone->tags }}"
                     id="ticketMilestoneChange{{ $ticket['id'] . $milestone->id }}"
```

Comment:
Just realized that this is going to break with multiple milestone selectors on the page. (like on kanban view) Can you make this a class or ensure the id is unique (using hash)?